R/tar-terra-rast.R

```diff
@@ -38,6 +44,7 @@ tar_terra_rast <- function(name,
                            pattern = NULL,
                            filetype = geotargets_option_get("gdal.raster.driver"),
                            gdal = geotargets_option_get("gdal.raster.creation.options"),
+                           preserve_metadata = FALSE, #TODO: add a geotargets option for this
```

Comment:
I think options are a good idea, we should add an issue for this so we can come back to it later

Reply:
Sounds great, but I don't think we can do this since the SpatRaster doesn't yet exist when the target object (including `tar_format()` with read/write functions) is created

Reply:
> I think options are a good idea, we should add an issue for this so we can come back to it later

I can implement it in this PR—I just forgot (exactly why I added a #TODO!)